neutral_generation/smart_convert.py

```diff
@@ -130,6 +130,17 @@ def simple_replace(token: Token):
         else:
             return None
 
+    # their vs theirs: https://ell.stackexchange.com/questions/18604/how-to-use-their-and-theirs
+    if text.lower() == 'his':
+        is_poss = (token.dep == poss and
```

Comment:
This variable name is confusing since both are possessive, right? Maybe check the opposite (that the head is not a noun) and call the variable poss_with_implied_head?

Reply:
Gotcha, I see what you mean. Will fix!

Reply:
One comment: I agree that both cases of "his" are possessive. However, I believe the dependency parser will usually mark the situation with the implied head as "attribute" or "determiner."

Example: 
![image](https://user-images.githubusercontent.com/36688734/90817519-c8f1d680-e2e2-11ea-8657-2c00f54920fc.png)

Reply:
To avoid this confusion, I've created a variable called "implied_head" to check whether the head is a noun or not.